Conversion-Class Temperatura

```diff
@@ -0,0 +1,16 @@
+package com.company;
+
+public class Temperatura {
+
+    public double temperatura;
+    public void setTemperatura(double temperaturaZilei) {
+        this.temperatura = temperaturaZilei;
+    }
+    public double getTemperatura(){
+        return temperatura;
+    }
+
+    public void conversionfromFtoC () {
+        double setTemperatura = 5/9*(this.temperatura-32);
```

Comment:
Variable names should not include verbs like `set` since that implies there is an action happening.
so rename your variable into `convertedTemperature`